### ISSUE TITLE
BGS Damage counter is incorrect in the theatre of blood

### ISSUE THREAD

**Issue by izaakgray** [runelite/runelite#9862]
XP modifiers are static so this should be fixable

**Comment by Nightfirecat** [comment]
From what I recall, we do not currently know the XP modifiers for parties of various sizes in ToB. If we can get that information, we can fix this issue.

**Comment by izaakgray** [comment]
Nylocas boss 1.1x
Sotetseg and bloat 1.5x

**Comment by TheStonedTurtle** [comment]
@izaakgray the xp modifiers aren't static as Nightfirecat mentioned: 
> we do not currently know the XP modifiers for parties of various sizes in ToB

The base stats of the NPC change depending on how many people are in the party and the xp modifier is based off the NPCs base stats. We'd need a monster examine of each boss with each possible party size to fix this issue.

**Comment by izaakgray** [comment]
Im pretty sure stats dont change based on scale, only HP does. Will see if i can gather some friendos to examine later however.

**Comment by TheStonedTurtle** [comment]
The HP effects the calculation too, as far as we know this is how jagex determines the xp modifier.
~~http://services.runescape.com/m=forum/c=PLuJ4cy6gtA/forums.ws?317,318,712,65587452,209,337584542#209~~

Apparently the forum post has been deleted, so here is the formula from one of my old PRs

```java
final double averageLevel = Math.floor((attackLevel + strengthLevel + defenceLevel + hitpoints) / 4);
final double averageDefBonus = Math.floor((stabDef + slashDef + crushDef) / 3);

return (1 + Math.floor(averageLevel * (averageDefBonus + bonusStrength + bonusAttack) / 5120) / 40);
```

**Comment by winterdaze** [comment]
People in the ToB community have already figured out the multipliers for various sizes: 
```
    SOTETSEG_5_MAN(1.675d, NpcID.SOTETSEG_8388),
    SOTETSEG_4_MAN(1.60d, NpcID.SOTETSEG_8388),
    SOTETSEG_3_MAN(1.525d, NpcID.SOTETSEG_8388),
    NYLOCAS_VASILIAS_5_MAN(1.2250d, NpcID.NYLOCAS_VASILIAS, NpcID.NYLOCAS_VASILIAS_8355, NpcID.NYLOCAS_VASILIAS_8356, NpcID.NYLOCAS_VASILIAS_8357),
    NYLOCAS_VASILIAS_4_MAN(1.20d, NpcID.NYLOCAS_VASILIAS, NpcID.NYLOCAS_VASILIAS_8355, NpcID.NYLOCAS_VASILIAS_8356, NpcID.NYLOCAS_VASILIAS_8357),
    NYLOCAS_VASILIAS_3_MAN(1.175d, NpcID.NYLOCAS_VASILIAS, NpcID.NYLOCAS_VASILIAS_8355, NpcID.NYLOCAS_VASILIAS_8356, NpcID.NYLOCAS_VASILIAS_8357),
    PESTILENT_BLOAT_5_MAN(1.85d, NpcID.PESTILENT_BLOAT),
    PESTILENT_BLOAT_4_MAN(1.7750d, NpcID.PESTILENT_BLOAT),
    PESTILENT_BLOAT_3_MAN(1.70d, NpcID.PESTILENT_BLOAT);
```

**Comment by whartd** [comment]
I'm not sure how many people would be willing to try soloing w/ monster examine thru Soteseg. Maybe @WooxSolo ?

**Comment by Adam-** [comment]
It doesn't scale below 3 people, so the above multipliers are correct I think.

**Comment by Adam-** [comment]
This is fixed in 8674a4f998ada7708de842ec98e9cc098b4c385c
